lib/games/true_false_game.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:jamaica/widgets/bento_box.dart';
+import 'package:jamaica/widgets/cute_button.dart';
+
+class _ChoiceDetail {
+  String choice;
+  Reaction reaction;
+  int index;
+
+  _ChoiceDetail({this.choice, this.reaction = Reaction.success, this.index});
+  @override
+  String toString() =>
+      '_ChoiceDetail(choice: $choice, reaction: $reaction, index: $index)';
+}
+
+class TrueFalseGame extends StatefulWidget {
+  final List<String> questions;
```

Comment:
We will have only one question. Not a list of questions. Also there is an optional answer. Not always. Again not a list. And there are no choices - only right or wrong